install/swiftly-install.sh

```diff
@@ -274,34 +332,54 @@ manually_select_platform () {
     echo "Please select the platform to use for toolchain downloads:"
 
     echo "0) Cancel"
-    echo "1) Ubuntu 22.04"
```

Comment:
I think we should update this selection to also allow the user to specify the actual platform string (e.g. "Ubuntu 20.04"). That way reordering the selections like this won't break any user scripts that might rely on a number being associated with a particular platform.

Reply:
Can't people already use `-p` to be sure of what platform to get

Reply:
Ah good point, yeah this is fine then.